test/tests/functional/pbs_hook_debug_input.py

```diff
@@ -113,3 +142,34 @@ def test_queuejob_hook_debug_input_has_queue_name(self):
         j2 = Job(TEST_USER, attrs=attr)
         self.server.submit(j2)
         self.match_queue_name_in_input_file(input_file_pattern, new_queue)
+
+    def test_mom_hook_debug_data(self):
+        """
+        Test that a debug enabled mom hook produces expected debug data.
+        """
+        def_que = self.server.default_queue
+        hname = "debug"
+        hook_body = """
+import pbs
+s = pbs.server()
+q = s.queue("%s")
+for vn in s.vnodes():
+    pbs.logmsg(pbs.LOG_DEBUG, "found vn=" + vn.name)
+pbs.event().accept()
+""" % (def_que,)
```

Comment:
You don't need to create a variable if you only use `self.server.default_queue` once. You also don't need to use tuple form for a single element.

Reply:
Yes, I forgot to remove that. I was referencing it in multiple places initially but changed that.

Reply:
I'll remove the tuple form, but I'm actually referencing self.server.default_queue also in line 172:
```
search = ["pbs.server().queue(%s).queue_type=Execution" % def_que]
```
And using the temporary variable allowed me to shorten the line to < 80 bytes to satisfy pep8.